packages/cms-drupal/src/Entity/Channel.php

```diff
@@ -102,6 +102,25 @@ public static function baseFieldDefinitions(EntityTypeInterface $entityType)
                     'type' => 'string_textfield',
                 ])
                 ->setDisplayConfigurable('form', true),
+            'domain' => BaseFieldDefinition::create('string')
+                ->setLabel(new TranslatableMarkup('Domain'))
+                ->setRequired(true)
+                ->setRevisionable(true)
+                ->setDefaultValueCallback(__CLASS__.'::getDefaultKeyValue')
+                ->setDisplayOptions('form', [
+                    'type' => 'string_textfield',
+                ])
+                ->setDescription('The domain name associated to this channel. e.g. www.example.com')
```

Comment:
Description is like the label, to be translatable it needs to use a translatable markup instead of a string.

Reply:
will do